src/Documentation/UserManual/DeviceOpticalMarkerTracker.dox

```diff
@@ -16,15 +16,18 @@ OpticalMarkerTracker is virtual device that provides position tracking of marker
 To get started, print a set of markers and create a configuration file. It is recommended to use "PlusServer: Optical marker tracker using MMF video" configuration file that is included in your Plus installation package (see also at the bottom of this page) and print this sheet that contains a number of markers with different sizes (make sure to print in actual size: scaling should be 100%):
 <a href="https://app.assembla.com/spaces/plus/subversion/source/HEAD/trunk/PlusLibData/ConfigFiles/OpticalMarkerTracker/marker_sheet_36h12.pdf?_format=raw">marker_sheet_36h12.pdf</a>. If you put the sheet in the field of view of your camera, the device will detect position and orientation of the markers.
 
-To improve tracking accuracy, you need to create a calibration file for your camera as described below. If you need more calibration markers, you can print more by following instructions below.
+To improve tracking accuracy, you need to create a calibration file for your camera as described below.
```

Comment:
Don't delete the entire sentence, I just meant _tracking_ markers instead of _calibration_ markers,